sst/weatherForecast/Weather_Forecast.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>Consider running <code>black</code> or <code>ruff</code> in this notebook to have the code formatting following standards. </p><p>Please, add docstrings to the functions.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/reports-performance-summary/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
I ran black and ruff to make modifications to my notebook.